Over21/views/AgeLimitSelectionView.swift

```diff
@@ -17,7 +17,7 @@ class AgeLimitSelectionView: UIView {
     
     // MARK: - Variables
     
-    public static var ageLimitThreshhold: Int = 21 // 21 by default
+    public static var ageLimitThreshhold: Int = 18 // 18 by default
```

Comment:
Please do not change the default Age limit.

Reply:
@EricGlaenzer 
long pan requested this change.
Will you check with him and let me know again?

Reply:
@ShawnDhanani 
We want to keep the default age limit to 21